src/components/Header.js

```diff
@@ -13,7 +13,7 @@ const Header = () => (
             <p>A go to place for investors and journalists to learn whether a given ICO is transparent and
                             can be trusted. The ICO Monitor serves as a transparency benchmark for the community to
                             better evaluate offerings and make informed decisions based on objective metrics.</p>
-            <a href="https://github.com/Neufund/ico-transparency-monitor#adding-custom-icos-to-the-transparency-monitor" target="_blank" className="add-new-ico"><i className="fa fa-plus" aria-hidden="true"/>Submit New ICO or Update HERE!</a>
+            <a href="https://github.com/Neufund/ico-transparency-monitor#adding-custom-icos-to-the-transparency-monitor" target="_blank" className="add-new-ico"><i className="fa fa-plus" aria-hidden="true"/>Submit New ICO or Update HERE! for free</a>
```

Comment:
This line is long I'm pretty sure it won't pass eslint. We don't need to fix all of eslint errors now but at least lets try not to introduce new ones.